react-native/index.d.ts

```diff
@@ -2563,8 +2563,18 @@ declare module "react" {
         ref?: Ref<DrawerLayoutAndroidStatic & ViewStatic>
     }
 
+    interface DrawerPosition {
```

Comment:
wow everything disappear :open_mouth:, maybe it's better to change drawerPosition to number, so it can match the type, I was reading the RN's doc when I face this problem.